### PR TITLE
metallb: add-frrtypes-schemes

### PR DESCRIPTION
**Pull request by gkopels**
Add frrk8 types schemes from "https://github.com/metallb/frr-k8s"

### CODE REVIEW

File: internal/sync/configs/frrk8-config.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (kononovn) [inline internal/sync/configs/frrk8-config.yaml:1]: You don't need to create new file. Simple add these lines to ./internal/sync/configs/metallb-config.yaml

File: internal/sync/configs/frrk8-config.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+- name: metallb-frr
+  sync: true
+  repo_link: "https://github.com/metallb/frr-k8s"
+  branch: main
+  remote_api_directory: api/v1beta1
+  local_api_directory: schemes/metallb/frrtypes
+
```
Comment (kononovn) [inline internal/sync/configs/frrk8-config.yaml:9]: remove empty line

Comment (gkopels) [reply internal/sync/configs/frrk8-config.yaml]: removed line

File: internal/sync/configs/frrk8-config.yaml (hunk 1 above)
Comment (gkopels) [reply internal/sync/configs/frrk8-config.yaml]: removed frr file and added to metallb

**Review by klaskosk — APPROVED** [review]
/lgtm

**Review by kononovn — APPROVED** [review]
/lgtm

**Pull request merged**